queries.sql

```diff
@@ -8,3 +8,11 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+SELECT COUNT(*) FROM animals;
```

Comment:
- [ ] please ensure to add steps when you `UPDATE` the animals' table by setting the species column to unspecified, then roll back the change
- [ ] Also steps when you commit a transaction, delete all records in the animals table, then roll back the transaction